src/init/omp.lua

```diff
@@ -60,12 +60,19 @@ function p:rightfilter(prompt)
     if tip_word == nil then
         return get_posh_prompt(true), false
     end
-    local prompt_exe = string.format('::OMP:: --config="::CONFIG::" --command="%s"', tip_word)
+    local prompt_exe = string.format('::OMP:: --shell=cmd --config="::CONFIG::" --command="%s"', tip_word)
     tooltip = io.popen(prompt_exe):read("*a")
     if tooltip ~= nil and tooltip ~= "" then
         return tooltip, false
     end
 end
+function p:transientfilter(prompt)
+    prompt = io.popen('::OMP:: --shell=cmd --config="::CONFIG::" --print-transient'):read("*a")
+    return prompt
```

Comment:
If Clink has transient prompt turned on but the theme doesn't define a transient prompt, then an empty string is used, which causes the prompt to malfunction (e.g. <kbd>Enter</kbd> by itself erases the prompt and draws a new prompt in the same spot, making the key appear to have no effect).

Inserting the following between lines 70 and 71 (i.e. immediately preceding the `return prompt`) appears to fix that:

```lua
    if prompt == "" then
        prompt = nil
    end
```